index.php

```diff
@@ -221,7 +221,7 @@ function smf_main()
 			require_once($sourcedir . '/LogInOut.php');
 			return ($_REQUEST['action'] == 'login2' ? 'Login2' : ($_REQUEST['action'] == 'logintfa' ? 'LoginTFA' : 'Logout'));
 		}
-		// Don't even try it, sonny.
+		// Don't even try it, sorry.
```

Comment:
!!!

Reply:
want you your sonny back?

Reply:
yes, please. I didn't write it, but it seems quite humorous.

...maybe that comment only makes senses in English-speaking countries?

Reply:
> Don't even try it, sorry

That seems like broken English to me.

Reply:
When you look into 2.0 code you'll find sonny ther, too.